### PR TITLE
Melee/Hunter initial loot prio BT

### PR DESCRIPTION
**Pull request by dillmcpickle**

### CODE REVIEW

**Review by ENFMAZZO — APPROVED** [review]

File: BT.json [hunk 1]
```diff
@@ -505,7 +505,7 @@
 			"wowID":"31101"
 		},
 		"Pauldrons of the Forgotten Protector":{
-			"prio":["Hunter/DPS War > Ele/Resto sham","EP/GP"],
+			"prio":["Chiron (1)","Dillmcpickle (1)","Thotpatrol (1)","Ferbb (1)","Oglol (2)","Putitah (2)","Paintrain (2)","Hunter/DPS War > Ele/Resto sham","EP/GP"],
```
Comment (zhang165) [inline BT.json:508]: Prot warrior first, the DPS shoulders are bis for threat while trading 10 stam for 100+ additional armor.

File: BT.json [hunk 2]
```diff
@@ -557,7 +557,7 @@
 			"wowID":"31098"
 		},
 		"Leggings of the Forgotten Protector":{
-			"prio":["Tatertotem(1)","Hunter/DPS War > Ele/Resto sham","EP/GP"],
+			"prio":["Tatertotem(1)","Chiron (1)","Doochie? (2)","Oglol (2)","Putitah (2)","Hunter/DPS War > Ele/Resto sham","EP/GP"],
```
Comment (zhang165) [inline BT.json:560]: Prot warrior first, most lists have t6 tank legs as bis.

**Review by zhang165 — CHANGES_REQUESTED** [review]

**Review by ENFMAZZO — APPROVED** [review]

File: BT.json (hunk 2 above)
Comment (dillmcpickle) [reply BT.json:560]: im fine with prot warrs getting the legs first but that means tater wont be getting the first 2 piece which is insane for ele.

File: BT.json (hunk 1 above)
Comment (dillmcpickle) [reply BT.json:508]: if you feel strongly about shoulders we can get your warrior an earlier shoulder token, but i dont think its a big enough of an upgrade of T5 prot shoulders to not give to the dps when its one of the biggest pieces hunters/warriors get all tier. ex rushing hunters on 2pc is massive, same with warriors for overall dmg. theres so much tanking gear for free for the prot warr, and if you need a tank with t6 theres a druid and pally that can MT basically every fight. and they will have their 4/5 set as soon as it drops.

Comment (zhang165) [reply BT.json:508]: Spoke offline, let's take a look after progression and see if threat is an issue.  I'm fine with this for now.

File: BT.json (hunk 2 above)
Comment (zhang165) [reply BT.json:560]: Yeah let's do that, tater will understand gearing a tank for mit is more important overall.

**Review by zhang165 — APPROVED** [review]

**Review by dillmcpickle — COMMENTED** [review]
snydi and thedruiid to T6 legs

**Pull request merged**